tf/train.py

```diff
@@ -158,21 +158,20 @@ def main(cmd):
     train_ratio = cfg['dataset']['train_ratio']
     experimental_parser = cfg['dataset'].get('experimental_v5_only_dataset',
                                              False)
-    num_train = int(num_chunks * train_ratio)
-    num_test = num_chunks - num_train
-    if 'input_test' in cfg['dataset']:
-        train_chunks = get_latest_chunks(cfg['dataset']['input_train'],
-                                         num_train, allow_less)
-        test_chunks = get_latest_chunks(cfg['dataset']['input_test'], num_test,
-                                        allow_less)
-    else:
-        chunks = get_latest_chunks(cfg['dataset']['input'], num_chunks,
-                                   allow_less)
-        if allow_less:
-            num_train = int(len(chunks) * train_ratio)
-            num_test = len(chunks) - num_train
+    if 'input' in cfg['dataset']:
```

Comment:
makes more sense to check for 'input' first than an arbitrary one of 'input_train' and 'input_test'